### PR TITLE
Implementation of Main.kt, Request.kt, according to architecture from A3

### PR DESCRIPTION
**Pull request by micahdbak**
This PR implements Request.kt, and Main.kt menu interaction.

Major notes:
- @ebarylko , my implementation doesn't use your Screen, Terminal, ..., interfaces because I feel we can achieve the same/similar things in less code, I want to hear back from you + the team on this

### CODE REVIEW

**Comment by ebarylko** [comment]
Please make sure that the code can be built before you push.

File: src/main/kotlin/anttracker/Main.kt [hunk 1]
```diff
@@ -5,39 +5,38 @@ Rev 1 - 2024/07/01 Original by Micah
 entry-point and main-menu of AntTracker.
  */
 
-package anttracker
```
Comment (ebarylko) [inline src/main/kotlin/anttracker/Main.kt:8]: Can you please clarify why main should not be in `anttracker`?

**Comment by ebarylko** [comment]
I am not clear what the pull request is about and why many other files are changed. Could you please add a summary adding what the PR is about and also the list of changed files and why they were modified? For reference see #29.

File: src/main/kotlin/anttracker/Main.kt (hunk 1 above)
Comment (micahdbak) [reply src/main/kotlin/anttracker/Main.kt:8]: I don't have a reason for this - I am not entirely sure how Kotlin's package management works.

**Comment by micahdbak** [comment]
@ebarylko , you can see which files are changed by clicking on the "Files Changed" tab. This PR reverts #25 , then makes my changes for Request.kt. That's all.

File: src/main/kotlin/anttracker/Main.kt (hunk 1 above)
Comment (ebarylko) [reply src/main/kotlin/anttracker/Main.kt:8]: The `package` keyword states what directory the file is located in.

**Comment by micahdbak** [comment]
[33](https://github.com/AntTracker/AntTracker/pull/33) accomplished this.

**Pull request closed without merging**